dbms/src/Server/Server.cpp

```diff
@@ -1094,12 +1090,21 @@ int Server::main(const std::vector<std::string> & /*args*/)
     /// Initialize main config reloader.
     auto main_config_reloader = std::make_unique<ConfigReloader>(
         config_path,
-        [&](ConfigurationPtr config) {
+        [&](ConfigurationPtr config, bool config_object_updated) {
+            LOG_DEBUG(log, "run main config reloader, config_object_updated: {}", config_object_updated);
             buildLoggers(*config);
             global_context->setMacros(std::make_unique<Macros>(*config, "macros"));
             global_context->getTMTContext().reloadConfig(*config);
             global_context->getIORateLimiter().updateConfig(*config);
             global_context->reloadDeltaTreeConfig(*config);
+            bool updated = global_context->getSecurityConfig()->update(*config); // Whether the cert path is updated.
```

Comment:
I think `global_context->getSecurityConfig()->update(*config)` itself can return true if `config_object_updated` is true, so we don't need to add the extra parameter?